.github/workflows/build-dev.yml

```diff
@@ -0,0 +1,30 @@
+name: Test
+on:
+  pull_request:
+    branches:
+      - develop
```

Comment:
Do you want this to run on PR to master also?

Reply:
I think I could probably have it run on both yes. I did plan to have publish in here too which is why it was limited to only dev. Will update now